### PR TITLE
fix code quality issues

### PR DESCRIPTION
**Pull request by withshubh**
#### Description
This PR fixes a few issues that were affecting the code quality.

#### Summary of changes
- Use literal syntax instead of function calls to create data structure
- Remove length check in favour of truthiness of the object

### CODE REVIEW

File: src/app/integrations/mailchimp/client.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def mass_subscribe(self, list_id: str, members: Iterable[MailchimpMember]):
                 'update_existing': True,
             },
         )
-        if len(response['errors']):
+        if response['errors']:
```
Comment (f213) [inline src/app/integrations/mailchimp/client.py:43]: Unfortunately this change brakes [this test](https://github.com/f213/education-backend/blob/master/src/app/tests/mailchimp/tests_mailchimp_mass_subscribe.py#L13). You may check it locally by running `make test`.
```suggestion
        if 'errors' in response and len(response['errors']):
```

**Comment by withshubh** [comment]
Hi @f213 :wave: 

I have reverted the fix in that particular file.

**Comment by f213** [comment]
Merged! Welcome and congrats with the first contribution 👍

**Comment by withshubh** [comment]
> Merged! Welcome and congrats with the first contribution

@f213 

I would like to inform you that I ran [DeepSource](https://deepsource.io/) analyzer on my fork of this repository to detect these issues. Have a look at the issues caught in this repository by DeepSource [here](https://deepsource.io/gh/withshubh/education-backend/issues/?category=recommended).

DeepSource is a code review automation tool that detects code quality issues and helps you to automatically fix some of them. You can use DeepSource to track test coverage, Detect problems in Dockerfiles, etc. in addition to detecting issues in code.

Can I submit a new PR with the configuration file which I used to configure the analysis?
You can merge the configuration file to continuously analyze the repo and fix code quality issues.

**Comment by f213** [comment]
> Can I submit a new PR with the configuration file which I used to configure the analysis?

Sure!

**Pull request merged**